tests/solana_tests/events.rs

```diff
@@ -78,6 +97,10 @@ fn less_simple_event() {
             function go() public {
                 emit MyOtherEvent(-102, "foobar", [55431, 7452], S({ f1: 102, f2: true}));
             }
+
+            function selector() public returns (bytes8) {
+                return MyOtherEvent.selector;
```

Comment:
Are we calling it selector even when we refer to discriminators on Solana?

Reply:
That's what's called in solidity. We also use `function.selector`